package.json

```diff
@@ -467,6 +467,11 @@
                     "default": null,
                     "description": "Number of threads to use for Julia processes."
                 },
+                "julia.highlightCurrentBlock": {
+                    "type": "boolean",
+                    "default": true,
```

Comment:
when I implemented "highlight current code block" feature in Juno, I found it a bit noisy to me imho (so I didn't push that anywhere remote). so I guess it might be better if this defaults to `false` but not sure. ppl may love this.

Reply:
Yeah, I think I'd turn it off pretty quickly myself. I'm slightly wary of implementing new features that are disabled from the off as I think they're then likely to be largely ignored. It would be good to get a bit of a consensus view on this (and that approach more broadly)